README.md

```diff
@@ -317,6 +317,78 @@ The content of the "file.csv" should be
 "Porsche",30000,"aqua"
 ```
 
+### Example 8
+
+You can also unwind arrays multiple times or with nested objects.
+
+```javascript
+var json2csv = require('json2csv');
+var fs = require('fs');
+var fields = ['carModel', 'price', 'items.name', 'items.color', 'items.items.position', 'items.items.color'];
+var myCars = [
+  {
+      "carModel": "BMW",
```

Comment:
Could you make the spacing 2 spaces instead of the 4 or tabs that it currently is. Just so it's consistent with the other examples. Thanks!